### PR TITLE
chore(doc): update readme. ws project management commandline

### PR DESCRIPTION
**Pull request by saitofun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,6 +269,43 @@ the query result like below:
...
+### Pub message and retrieve ZKP by `ioctl`
+
+```sh
+ioctl ws message send --project-id ${project id} --project-version ${project version} --data ${message content}
+ioctl ws message query --message-id ${message id}
+```
+
+### w3bstream project management
+
+You need deploy project register contract to **IOTX** before operating w3bstream projects.
```
Comment (huangzhiran) [inline README.md:281]: maybe we can give users contract address we have deployed, and users  just write the contract and don't need deploy the contract themself

File: README.md [hunk 2]
```diff
@@ -250,6 +269,43 @@ the query result like below:
 }
 ```
 
+### Pub message and retrieve ZKP by `ioctl`
+
+```sh
+ioctl ws message send --project-id ${project id} --project-version ${project version} --data ${message content}
```
Comment (huangzhiran) [inline README.md:275]: change all wsctl and use ioctl to replease ?

Comment (saitofun) [reply README.md:275]: remove `wsctl` after all commands migrating to `ioctl`?

File: README.md (hunk 1 above)
Comment (saitofun) [reply README.md:281]: the `project register contract address` is a command parameter supporting testing.

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**